compat/operator/takeWhile.ts

```diff
@@ -34,9 +34,13 @@ import { takeWhile as higherOrder } from 'rxjs/operators';
  * @return {Observable<T>} An Observable that emits the values from the source
  * Observable so long as each value satisfies the condition defined by the
  * `predicate`, then completes.
+ * @param {boolean} inclusive When set to `true` the value that caused
+ * `predicate` to return `false` will also be emitted.
  * @method takeWhile
  * @owner Observable
  */
-export function takeWhile<T>(this: Observable<T>, predicate: (value: T, index: number) => boolean): Observable<T> {
-  return higherOrder(predicate)(this);
+export function takeWhile<T>(this: Observable<T>,
+                             predicate: (value: T, index: number) => boolean,
+                             inclusive?: boolean): Observable<T> {
+  return higherOrder<T>(predicate, inclusive)(this);
```

Comment:
Because this is a new feature, we don't need to update the backward compatibility code. You can revert this file.